src/commons/release/__init__.py

```
@@ -0,0 +1,8 @@
+"""commons.release - OpenShift release-controller helpers."""
+
+from commons.release.client import (
+    RELEASE_CONTROLLER_API,
+    ReleaseControllerClient,
+)
+
+__all__ = ["ReleaseControllerClient", "RELEASE_CONTROLLER_API"]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)